go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/iancoleman/strcase v0.2.0
 	github.com/martinlindhe/base36 v1.1.1
 	github.com/open-policy-agent/opa v0.54.0
+	github.com/pomerium/csrf v1.7.0
```

Comment:
This change really shouldn't affect the go.mod file, should it?

Reply:
I have zero idea why this package keeps jumping between direct and indirect requirements sections

Reply:
actually I now remembered why :-)